test/functional/test_framework/util.py

```diff
@@ -183,7 +183,7 @@ def check_json_precision():
 def count_bytes(hex_string):
     return len(bytearray.fromhex(hex_string))
 
-def hash256(byte_str):
+def hash256r(byte_str):
     sha256 = hashlib.sha256()
     sha256.update(byte_str)
     sha256d = hashlib.sha256()
```

Comment:
Looks like you can replace all of this code with `hash256()[::-1]`

Reply:
yeah but then there will be a recursive imports. (messages already imports util, this will require util to also import messages)

But if I move this function to zmq tests then that's possible